pom.xml

```diff
@@ -928,16 +924,6 @@
         <version>${netty.version}</version>
         <classifier>osx-x86_64</classifier>
       </dependency>
-      <!-- SPARK-38541: Add the explicit dependency on `netty-tcnative-classes` because the dependency on
-           `netty-tcnative-classes` is changed to `Optional` in [netty](https://github.com/netty/netty/pull/12146).
-           `netty-tcnative-classes` module is mainly used by SSL package in `netty-handler`.
```

Comment:
Netty support openSSL and JDK SSL, to enable openSSL support `netty-tcnative-classes` and platform native libs should be existed in classpath.
Only `netty-tcnative-classes` exist will cause https://github.com/netty/netty/issues/12101